src/components/Filters/FilterRow/FilterRow.tsx

```diff
@@ -123,7 +124,11 @@ const FilterRow: React.FC<FilterRowProps> = ({
   };
 
   const handleInputChange = (event) => {
-    onInputChange(event.target.value, index);
+    if (event.target) {
```

Comment:
Why do we need to split this in condition?

Reply:
It is generic handler
- if input component is `Input` then event is Event type
- if input component is DateRangePicker then event is object `{startDate, endDate}`
- if input component is Select then event is object `{value, label}` - not yet implemented
- if input component is SingleDatePicker then event is date - not yet implemented

Maybe event is misleading name now, I can change it. Also the condition will change when I will add other components.

Reply:
ah ok I was misled by the name

Reply:
Ok, I will rename it in next PR